docs/CHANGELOG.asciidoc

```diff
@@ -30,6 +30,223 @@ This section summarizes the changes in each release.
 
 coming[7.3.0]
 
+See also <<breaking-changes-7.3, breaking changes in 7.3>>.
+
+[float]
+[[enhancement-7.3.0]]
+=== Enhancements
+Canvas::
+* Feat: Keyboard shortcuts for nudging elements {pull}39208[#39208]
+* [Fixes #27123]  Feat: Zoom In/Out {pull}38832[#38832]
+* Add joinRows Canvas expression function {pull}38680[#38680]
+* [Fixes #27124]  Feat: Fit Workpad to Window {pull}39864[#39864]
+* [Fixes #23061, #23144, #29526] Enable Canvas URL Parameters {pull}40061[#40061]
+* Alignment and distribution {pull}39132[#39132]
+* [Fixes #30841]  Feat: Box select {pull}32995[#32995]
+Code::
```

Comment:
Since Code is turned off by default, I think we should either remove this section altogether or put in a disclaimer that Code is not enabled in default installation....